lib/App/ModuleBuildTiny/Dist.pm

```diff
@@ -86,7 +86,19 @@ sub distname {
 }
 
 sub detect_license {
-	my ($data, $filename, $authors) = @_;
+	my ($data, $filename, $authors, $mergedata) = @_;
+	if ($mergedata->{license} && @{$mergedata->{license}} == 1) {
+		require Software::LicenseUtils;
+		Software::LicenseUtils->VERSION(0.103014);
+		my $spec_version = $mergedata->{'meta-spec'} && $mergedata->{'meta-spec'}{version} ? $mergedata->{'meta-spec'}{version} : undef;
```

Comment:
mbtiny assumes meta-spec 2.0 if no explicit spec is given, it should probably do the same here.